Tests/Unit/EmogrifierTest.php

```diff
@@ -1766,6 +1766,19 @@ public function removeExcludedSelectorGetsMatchingElementsToBeEmogrifiedAgain()
         self::assertContains('<p class="x" style="margin: 0;"></p>', $result);
     }
 
+    /**
+     * @test
+     *
+     * @expectedException \InvalidArgumentException
+     */
+    public function emogrifyThrowsInvalidArgumentExceptionForInvalidExcludedSelectorInDebugMode()
```

Comment:
We also need a test for the non-debug case (that tests that no exception is caught, e.g. indirectly by checking that some particular HTML is returned).